src/file-utils.ts

```diff
@@ -0,0 +1,22 @@
+import * as fs from 'fs';
+import type { CommandContext } from './commands/prompt/commands';
+
+export const getDefaultFileName = (context: CommandContext) => {
+  const currentDate = new Date();
+  const today = currentDate.toISOString().split('T')[0];
```

Comment:
using toISOString will result in UTC timezone. We probably want to keep it in the user's local TZ.

Reply:
Done